### ISSUE TITLE
Long press on textbox with no keyboard attached shouldn't open the keyboard

### ISSUE THREAD

**Issue by MarcAnt01** [microsoft/microsoft-ui-xaml#3412]
# Proposal: [Don't open the keyboard with long press on  a textbox] 

## Summary

I don't think that long press should open the keyboard, just a tap should

## Rationale

* A user could simply want to paste a photo or some text, without modifying it through keyboard
* This is the behavior of Android and I think it's more intuitive rather Windows one. 

## Scope

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |
| This proposal will allow end users to accomplish X | Should |
| This proposal will allow developers to accomplish Y | Could |
| This proposal will allow end users to accomplish Z | Won't |

**Comment by StephenLPeters** [comment]
We open the soft keyboard whenever a text box gets focus, and by long pressing on the textbox you are putting focus on the text box. I think this is by design.  For example, if you wanted to long press text to highlight it all to delete it, how would you do that without the soft keyboard opening?

**Comment by MarcAnt01** [comment]
@StephenLPeters good point, but imagine the scenario where there is no text in the textbox, in this case I think it is a good idea not to open the keyboard, your thoughts? Also imagine a messaging app with a textbox to write a message, long pressing it will open the context menu first and the keyboard later, making the experience annoying

**Comment by MarcAnt01** [comment]
This "intended behavior" is bringing horrible results (part of the context menu is hidden), like this one, can this be reopened?
![image](https://user-images.githubusercontent.com/43090263/96370850-8ff9a500-115f-11eb-97b0-9f402e8282dc.png)

**Comment by StephenLPeters** [comment]
That does seem like a bug, the context menu should not open in the space reserved for the soft keyboard. But I think it is a different issue than long pressing not opening the keyboard. Are you suggesting repurposing this issue or that we should open another?

**Comment by MarcAnt01** [comment]
@StephenLPeters I am changing this issue to be a feature request and I will open another ticket for the mentioned bug, is that all right?

**Comment by StephenLPeters** [comment]
Sounds correct to me.

**Comment by MarcAnt01** [comment]
I have refactored the issue into a feature request

**Comment by MarcAnt01** [comment]
@StephenLPeters can you please reopen this ticket?
